IntelPresentMon/PresentMonService/CliOptions.h

```diff
@@ -26,8 +26,8 @@ namespace clio
 	private: Group gl_{ this, "Logging", "Control logging behavior" }; public:
 		Option<std::string> logDir{ this, "--log-dir", "", "Enable logging to a file in the specified directory" };
 		Option<std::string> logPipeName{ this, "--log-pipe-name", pmon::gid::defaultLogPipeBaseName, "Name of the pipe to connect to for log IPC" };
-		Flag disableStdioLog{ this, "--disable-stdio-log", "Disable logging to stderr" };
-		Flag disableDebuggerLog{ this, "--disable-debugger-log", "Disable logging to system debugger" };
+		Flag enableStdioLog{ this, "--enable-stdio-log", "Disable logging to stderr" };
```

Comment:
I am confused by the command line descriptions. The arg says enable and the description says disable. By enabling the option are you just supposed to know you are disabling something?